### ISSUE TITLE
[Organizer Request] Add Baris Sonmez to Amsterdam chapter

### ISSUE THREAD

**Issue by sbarissonmez** [cncf/communitygroups#164]
### City or location name for your CNCG

Cloud Native Amsterdam

### What country do you want to start your community group in?

Netherlands

### Tell us about yourself. List your name, email, company, GitHub URL, LinkedIn URL, and CNCF Slack workspace name (or if you need to be invited), and a required profile link on community.cncf.io

https://www.linkedin.com/in/sbarissonmez/

### Tell us about your co-organizer. List their name, email, company, GitHub URL, LinkedIn URL, and CNCF Slack workspace name (or if they need to be invited), and a required profile link on community.cncf.io

https://github.com/sbarissonmez

### Tell us about your co-organizer. List their name, email, company, GitHub URL, LinkedIn URL, and CNCF Slack workspace name (or if they need to be invited), and a required profile link on community.cncf.io

https://github.com/sbarissonmez

### What is your mission statement or objective for this new chapter?

We’re excited about transforming the future of technology through AI, cloud solutions, and advanced networking. Our focus is on the cutting-edge tools that drive innovation—whether it’s cloud-native architectures, multi-cloud strategies, or advanced networking protocols. Any level of expertise is welcome; we’re all navigating this evolving landscape together, and we’re committed to fostering an open, inclusive environment for technologists, leaders, and enthusiasts to learn, collaborate, and grow. Let’s build the future of tech—together.

### Code of Conduct

- [x] I/We agree to follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md)

### Where organizers reside

- [x] I/We confirm I/we am/are local to or live near the city of the above-proposed chapter.

### Organizers companies/affiliations

- [x] I/We agree if there is more than one organizer now or onboarded in the future, the majority come from different companies

### Inclusive

- [x] I/We agree. To assure this, I/we have recently completed the [Inclusive Open Source Community Orientation (LFC102) course](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)

### Vendor Neutral

- [x] I/We have read what [vendor-nuetrality means](https://contribute.cncf.io/maintainers/community/vendor-neutrality/), and agree that the main content of our meetups will remain vendor-nuetral

### Agree to use CNCG branding

- [x] I/We agree to use the CNCG logo in my/our chapter and social branding

### Registration through CNCF platforms

- [x] I/We agree to host free registration in the main CNCF platform

### Remain an active chapter

- [x] I/We agree to have a meetup at least once every 90 days, or else communicate reasoning for upcoming inactivity.

### Collaboration with other chapters

- [x] I/We acknowledge the CNCF right to approve additional chapters in my area based on this criteria

### Aim to have a diverse representation of speakers in your meetups

- [x] I/We will do our best to host diverse speakers throughout the year

**Comment by embik** [comment]
I'm not associated with the CNCF but there is already a community group for Amsterdam: https://community.cncf.io/amsterdam/, heads up.

**Comment by AudMonte01** [comment]
Yes, that is correct. Thank you, @embik.

@sbarissonmez - do you wish to join as an organizer? If so I can loop in the current organizers to review, and let you know the best steps.

**Comment by sbarissonmez** [comment]
Yes, I wish to join as an organizer. Could you give a little more details organizer responsibility and requirements, please? Thank you @AudMonte01

**Comment by AudMonte01** [comment]
Aside from the terms you agreed to above, here is a [Best Practices guide](https://github.com/cncf/communitygroups/blob/main/best_practices.md) that might be helpful, as well as the [ReadMe](https://github.com/cncf/communitygroups/blob/main/README.md).

**Comment by AudMonte01** [comment]
I am copying @ams0 and @wrkode to let you know best steps. If you have not already, please first participate in [this group's meetups](https://community.cncf.io/amsterdam/) as a consistent member.

**Comment by ams0** [comment]
Thank you for your interest @sbarissonmez , feel free to reach out in the #netherlands channel or privately on CNCF Slack. We do have a requirement that you must live in The Netherlands (for a significant amount of a year's time) to be active in our organiser team; if you have just moved, that's great and let's get you onboarded!

**Comment by AudMonte01** [comment]
Thank you for your response, @ams0 

@sbarissonmez - kindly respond about the location you reside in. Please note, CNCF typically recommends you join as a member, and consistently attend meetups ahead of becoming as a co-organizer.

You can do so here: https://community.cncf.io/cloud-native-amsterdam/
